WordPress/Classes/Services/PostCoordinator.swift

```diff
@@ -50,31 +54,44 @@ class PostCoordinator: NSObject {
     ///
     private func uploadMedia(for post: AbstractPost, automatedRetry: Bool = false) -> Bool {
         let mediaService = MediaService(managedObjectContext: backgroundContext)
-        let failedMedia: [Media] = post.media.filter({ $0.remoteStatus == .failed })
-        let mediasToUpload: [Media]
+        let media: [Media]
```

Comment:
⚠️ I'm not sure I understand how these changes in `uploadMedia` are related to this PR. It looks like this was changed during dddc899e692666d3f854d9b6dc4669f348e4ed43. Would you mind explaining more?

Reply:
This was the wrong change. This shouldn't have been modified here, reverted.